test/spec/features/stack/install_spec.rb

```diff
@@ -64,4 +64,21 @@
       expect(k.out.match(/stop_grace_period: 23s/)).to be_truthy
     end
   end
+
+  context 'For a stack with read_only' do
+    it 'creates stack service with read_only and updates it properly' do
+      with_fixture_dir("stack/read_only") do
+        run 'kontena stack install redis.yml'
+      end
+      k = run 'kontena service show simple/redis'
+      expect(k.code).to eq(0)
+      expect(k.out.match(/read_only: yes/)).to be_truthy
+      with_fixture_dir("stack/read_only") do
+        run 'kontena stack upgrade simple redis_read_only_false.yml'
+      end
+      k = run 'kontena service show simple/redis'
```

Comment:
A `kontena container inspect` to look for `HostConfig.ReadonlyRootfs: true` would be even better (more end-to-end)